adr/2024-11-2-assertion-format.md

```diff
@@ -1,78 +1,101 @@
-# [short title of solved problem and solution]
+# `format` as an assertion
```

Comment:
This is a different ADR and shouldn't be changing in this PR?

Reply:
There were some formatting things that were merged and failing on the main branch and causing this build to fail.  Jason and I just cleaned it up here.